spec/operators/publishReplay-spec.ts

```diff
@@ -23,6 +23,34 @@ describe('Observable.prototype.publishReplay', () => {
     expect(source instanceof Rx.ConnectableObservable).toBe(true);
   });
 
+  it('should follow the RxJS 4 behavior and NOT allow you to reconnect by subscribing again', (done: DoneSignature) => {
```

Comment:
simple test case reordering again, due to jasmine behavior. It seems occuring repeatedly now.